<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormReplay</class>
 <widget class="QWidget" name="FormReplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>回放</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/Image/Chess/图标.jpg</normaloff>:/new/prefix1/Image/Chess/图标.jpg</iconset>
  </property>
  <widget class="QLabel" name="background_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/Gme1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="ChessBoard" name="checkerboard_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>840</width>
     <height>840</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Background/check.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next_Button">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>700</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/100.40.jpg);
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="last_Button">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>700</y>
     <width>105</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/100.40.jpg);
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_Button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>800</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openboard_button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>200</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>打开棋局</string>
   </property>
  </widget>
  <widget class="QComboBox" name="playPattern_combox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>400</y>
     <width>155</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Image/Button/150.50.jpg);
selection-background-color: rgb(41, 40, 100);
selection-color: rgb(255, 255, 255);
color: rgb(255, 85, 255);
</string>
   </property>
   <item>
    <property name="text">
     <string>自动</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>手动</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>500</y>
     <width>155</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Image/Button/150.50.jpg);
color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="speed_SpinBox">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>600</y>
     <width>155</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Image/Button/100.40.jpg);
color: rgb(255, 170, 255);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="prefix">
    <string>速度:</string>
   </property>
   <property name="suffix">
    <string>倍</string>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChessBoard</class>
   <extends>QLabel</extends>
   <header location="global">chessboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Replay.qrc"/>
 </resources>
 <connections/>
</ui>
